src/plumbing/transcoding.c

```diff
@@ -1010,8 +1010,15 @@ transcoder_stream_video(transcoder_t *t, transcoder_stream_t *ts, th_pkt_t *pkt)
       octx->qmin           = 1;
       octx->qmax           = FF_LAMBDA_MAX;
 
-      octx->bit_rate       = 2 * octx->width * octx->height;
-      octx->rc_max_rate    = 4 * octx->bit_rate;
+      if (t->t_props.tp_vbitrate == 0) {
+        octx->bit_rate       = 2 * octx->width * octx->height;
+        octx->rc_max_rate    = 4 * octx->bit_rate;
+
+      } else {
+        octx->bit_rate       = t->t_props.tp_vbitrate * 1000;
+        octx->rc_max_rate    = octx->bit_rate * 1.05;
```

Comment:
I have changed alot since this PR, and I will update it when I have cleaned it up, added some other features, and testing.

I have set rc_max_rate = bit_rate since it is intended as a bitrate limiter, there is no limit on lower.
Anyone who wants quallity can now set the qmin quatizer instead of bitrate.